app/client/src/components/isochrones/Isochrones.vue

```diff
@@ -660,9 +660,11 @@
                             <v-icon small class="text-xs-center mx-2"
                               >fas fa-clock
                             </v-icon>
-                            <span class="caption">{{
-                              calculation.config.settings.weekday
-                            }}</span>
+                            <span class="caption"
+                              >{{ publicTransport.fromTime }}
+                              -
+                              {{ publicTransport.toTime }}</span
+                            >
```

Comment:
This should come from the calculation object as it should not change when the user selects a new time range. 
The method `secondsToHoursAndMins` can directly be called here 
e.g. `secondsToHoursAndMins(calculation.config.settings.from_time)`
To use the function in the template, you should add it to the methods object like below: 

```
 methods: {
    secondsToHoursAndMins, <-----
    renderRadarChart: function() {
      let labels = this.selectedPoisOnlyKeys;
      let labels;
      if (this.chartDatasetType === 1) {
     ....................

```

Reply:
ok, I will do the changes